### DISCUSSION TITLE
== Error ==  file not found: ../reference/GCF_000005845.2_ASM584v2_genomic.fna$ (single reads, library number: 1, library type: trusted-contigs)

### DISCUSSION THREAD

**Discussion by wagaw12** [ablab/spades#1203] [General]
I was doing the assembly using -- isolate and -- trusted contings options and I have the problem of running it successfully, as the log file shows the above problem. Any help?

**Comment by asl** [comment]
Per error message – file was not found. Are you sure its filename contains an `$` sign?

**Reply by wagaw12** [reply]
Thanks; it has no $ sign, and it works as it should.
